packages/diagrams/frontend/sirius-components-diagrams/src/dialog/diagramDialogExtensionPoint.types.ts

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { GQLToolVariable } from '../renderer/palette/Palette.types';
+export interface DiagramDialogContribution {
+  canHandle: (dialogTypeId: string) => boolean;
```

Comment:
Why not use the descriptionId? Why have two pieces of information?

Reply:
removed and replaced by the descriptionId that now contains the type information.